keps/sig-node/2129-remove-cadvisor-json-metrics/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Disable cAdvisor json Metrics
+kep-number: 2129
+authors:
+  - "@dashpole"
+owning-sig: sig-node
+participating-sigs: [sig-instrumentation]
```

Comment:
Hi there, 1.21 Enhancements Lead here. 
Is this owned by node or instrumentation? This enhancement was opted in by Sig Instrumentation therefore, wanted to double-check and make sure the tracking sheet reflects the correct SIG as the owner.

cc @ehashman

Reply:
Node is the owning SIG, Instrumentation is participating, Node has to approve and SIG Instrumentation gives our +1